beacon_node/beacon_chain/src/historical_blocks.rs

```diff
@@ -127,6 +128,12 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
                 self.store
                     .blobs_as_kv_store_ops(&block_root, blobs, &mut blob_batch);
             }
+            // Store the data columns too
+            if let Some(data_columns) = maybe_data_columns {
+                // new_oldest_data_column_slot = Some(block.slot());
+                self.store
+                    .data_columns_as_kv_store_ops(&block_root, data_columns, &mut blob_batch);
```

Comment:
I think we may want a `Vec<KeyValueStoreOp>` dedicated to data_columns instead of re-using `blob_batch`

Reply:
I think the benefit I see with using a dedicated one is if we want to write to a separate database? I'm re-using the blob db for simplicity here (since the maintainence/pruning logic are the same), but happy to create a separate db if there's good reasons to do that.

Reply:
So I'm reusing the same blobs db for data columns but using a different `DbColumn` to separate data columns from blobs.

It feels like we could benefit from better UX - no change for users that's already using the `blobs-db` flag and save on some code duplication.